snapcraft/cli/_errors.py

```diff
@@ -55,3 +71,15 @@ def exception_handler(exception_type, exception, exception_traceback, *,
             echo.error(str(exception))
 
     sys.exit(exit_code)
+
+
+def _submit_trace(exception):
+    client = RavenClient(
+        'https://b0fef3e0ced2443c92143ae0d038b0a4:'
+        'b7c67d7fa4ee46caae12b29a80594c54@sentry.io/277754',
```

Comment:
Curious, how can we prevent to be spammed with evil messages to our sentry?

Reply:
I don't think we can.